codice/ddf
# DDF-2316: Fixes unit tests on Windows.

#### What does this PR do?
Fixes the unit tests on Windows. 

#### Who is reviewing it?
@tbatie 
@adimka 

#### Choose 2 committers to review/merge the PR
@coyotesqrl
@shaundmorris

#### How should this be tested?
Verify build on your local Mac. 
Run a Jenkins or VM build on Windows and ensure unit tests pass. 

##### Summary for unit test fixes:
* Resolves AbstractExpansionTest failure on Windows.
* Consolidated test setup in HazelcastNotificationStoreTest.
* Run the path to Hazelcast xml config through AbsolutePathResolver.
* Update FileSystemPersistenceProvider, TestZipValidator, and TestZipCompression.
* ResourceCacheImplTest was not cleaning up after itself.
* ResourceCacheImplSizeLimitTest was deleting resources on every run.
* TestDumpCommand was incorrectly delimiting expected console output.
